force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -218,14 +285,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createIntegerList(Integer n) {
         // Create a new list of integers 
-
-        /*
-        for() {
-            //add integer to the list
+        ​Integer n = 5; 
+        List<Integer> numberList = new List<Integer>{};
+            for(Integer i = 1; i <= n; i++) {
+                numberList.add(i);
+                System.debug(numberList);
         }
```

Comment:
Again don't hard code the value for n. This method should work with any number. Not just 5.  What happens when someone calls createIntegerList(10) ?